src/server/cluster/cluster_family.cc

```diff
@@ -494,12 +494,35 @@ void ClusterFamily::DflyClusterConfig(CmdArgList args, ConnectionContext* cntx)
 
   lock_guard gu(set_config_mu);
 
+  RemoveOutgoingMigrations(new_config->GetFinishedOutgoingMigrations(tl_cluster_config));
+  RemoveIncomingMigrations(new_config->GetFinishedIncomingMigrations(tl_cluster_config));
+
+  SlotRanges enable_slots, disable_slots;
+  {
+    std::lock_guard lk(migration_mu_);
+    // If migration state is changed simultaneously, the changes to config will be applied after
+    // set_config_mu is unlocked and even if we apply the same changes 2 times it's not a problem
+    for (const auto& m : incoming_migrations_jobs_) {
+      if (m->GetState() == MigrationState::C_FINISHED) {
+        const auto& slots = m->GetSlots();
+        enable_slots.insert(enable_slots.end(), slots.begin(), slots.end());
```

Comment:
```suggestion
        enable_slots = m->GetSlots();
```
Can we do this (also below)? They seem to be of the same type, am I missing something?

Reply:
Sorry, my bad, please ignore. I thought we could just assign, but we're inside a `for` loop so we need to append to the vector...